runtime/rococo/src/weights/frame_system.rs

```diff
@@ -43,8 +43,10 @@ use sp_std::marker::PhantomData;
 /// Weight functions for `frame_system`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> frame_system::WeightInfo for WeightInfo<T> {
-	fn remark(_b: u32, ) -> Weight {
+	fn remark(b: u32, ) -> Weight {
```

Comment:
Weird that this is not ignore here.   
For all other runtimes it also is the case.

Reply:
`1_000` is literally the smallest value that can be returned from the benchmarking tool. (representing 1 NS)

This is just noise, and potentially we could round these things down to zero, but I see no problem with there being something here, and it being slightly inconsistent. It is negligible.